debian/rules.d/2-binary-arch.mk

```diff
@@ -165,6 +165,15 @@ endif
 			echo "$$dtb_file ?" >> $(DEBIAN)/d-i/firmware/$(arch)/kernel-image; \
 		done; \
 	fi
+
+	# Have generic uImage containing all dtbs
```

Comment:
please extend this comment a little more, to explain the overall reason of why we are doing this

Reply:
Extended the explanation in new commit.